docs/Makefile

```diff
@@ -4,7 +4,7 @@ VOLUMES := -v $(ROOT_DIR):/src
 HUGO_VERSION := 0.107.0
 IMAGE := klakegg/hugo:$(HUGO_VERSION)-ext
 PORT := 1314
-DOCKER_CMD := docker run --rm -t -e HUGO_CACHEDIR=/src/tmp/.hugo
+DOCKER_CMD := docker run --rm -t -e HUGO_CACHEDIR=/src/tmp/.hugo -e HUGOxPARAMSxGITHUB_REPO=""
```

Comment:
what is that doing?

Reply:
Hugo configuration overwrite. HTMLTest has issues if certain links are not working. specially problematic if we do generate new files, which don't exist in `main` - then our github edit links, etc. are broken (the nice little feature on the side).

But the rendering gets disabled when we set the config property `github_repo` to empty.

Now comes the tricky part. Hugo allows environment variable  overwrites, but sees the `_` as the normal separator for configuration levels so `HUGO_PARAMS_GITHUB_BRANCH` would resolve to 

```yaml
params:
  github:
    branch:
```
but we want `params.github_branch`. Hugos solution to this problem is, that the first character after `HUGO` in an environment variable is seen as the `delimiter`. Therefore to make variables with an underscore work, we need to change the delimiter :) - see https://github.com/keptn/lifecycle-toolkit/pull/1010 (which was the pr to fix this, we just wanted to unblock this pr, and added the change also here) and more specifically https://gohugo.io/getting-started/configuration/#configure-with-environment-variables